src/storage/index/LookUpEdgeIndexProcessor.cpp

```diff
@@ -0,0 +1,69 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "LookUpEdgeIndexProcessor.h"
+
+namespace nebula {
+namespace storage {
+
+void LookUpEdgeIndexProcessor::process(const cpp2::LookUpIndexRequest& req) {
+    /**
+     * step 1 : prepare index meta and structure of return columns.
+     */
+    auto ret = prepareRequest(req);
+    if (ret != cpp2::ErrorCode::SUCCEEDED) {
+        putResultCodes(ret, req.get_parts());
```

Comment:
onFinished

Reply:
When `putResultCodes` is called, onFinished() is included in this function.